py3status/modules/file_status.py

```diff
@@ -98,30 +104,33 @@ def file_status(self):
         paths = list(map(glob, self.path))
         # merge list of paths
         paths = [x for x in chain.from_iterable(paths)]
+        file_status_data['files'] = len(paths)
```

Comment:
`paths` is better. More explicit and the fact that this can pick up directories too.

Reply:
Yup